mxabierto/fonden
# Update map layer names and keys

### TODO
- [x] Update map layer names
- [x] Load GeoJSONs using leaflet
- [x] Compress large files using TopoJSON

### How to test

Select any option of the map toolbox, and you should be able to see some blue polygons appear and disappear on the map when you click.

<img width="890" alt="screen shot 2015-07-22 at 3 30 27 pm" src="https://cloud.githubusercontent.com/assets/705860/8836439/9bd9f24a-3086-11e5-8652-807824b36943.png">
